kitsune/wiki/models.py

```diff
@@ -742,7 +743,7 @@ def gets_mozilla_account_cta(self):
             self.parent.slug if self.parent else self.slug
         ) in settings.MOZILLA_ACCOUNT_ARTICLES
 
-    @property
+    @cached_property
```

Comment:
Nice! How about adding a method to do the calculation and then set that method here as a cached property? This way we can force also the evaluation if ever needed without having logic in the property. Eg

```python
def get_is_restricted(self):
     """Is this document restricted in terms of visibility?"""
        if self.parent:
            # Translations follow their parent's restrictions.
            return self.parent.is_restricted
        return self.restrict_to_groups.exists()

@ cached_property
def is_restricted(self):
    return self.get_is_restricted()
```

Reply:
That's a great idea, thanks!